README.md

```diff
@@ -0,0 +1,45 @@
+# Galoy CLI
+Galoy CLI is a Rust-based CLI client that can interact with the Galoy backend using GraphQL queries and mutations.
+
+## Installation
+To install Galoy CLI locally and set up a local environment:
+
+1. Clone the repository using git clone `https://github.com/GaloyMoney/galoy-cli.git`.
```

Comment:
1. Clone the /galoy-cli repository using `git clone https://github.com/GaloyMoney/galoy-cli.git`